instrumentation/src/main/java/io/opentelemetry/android/instrumentation/crash/CrashReporter.java

```diff
@@ -38,16 +45,47 @@ public void installOn(InstrumentedApplication instrumentedApplication) {
                 Thread.getDefaultUncaughtExceptionHandler();
         Thread.setDefaultUncaughtExceptionHandler(
                 new CrashReportingExceptionHandler(
-                        buildInstrumenter(instrumentedApplication.getOpenTelemetrySdk()),
-                        instrumentedApplication.getOpenTelemetrySdk().getSdkTracerProvider(),
+                        buildInstrumenter(
+                                instrumentedApplication
+                                        .getOpenTelemetrySdk()
+                                        .getSdkLoggerProvider()),
+                        instrumentedApplication.getOpenTelemetrySdk().getSdkLoggerProvider(),
                         existingHandler));
     }
 
-    private Instrumenter<CrashDetails, Void> buildInstrumenter(OpenTelemetry openTelemetry) {
-        return Instrumenter.<CrashDetails, Void>builder(
-                        openTelemetry, "io.opentelemetry.crash", CrashDetails::spanName)
-                .addAttributesExtractor(new CrashDetailsAttributesExtractor())
-                .addAttributesExtractors(additionalExtractors)
-                .buildInstrumenter();
+    private void emitCrashEvent(Logger crashReporter, CrashDetails crashDetails) {
+        Throwable throwable = crashDetails.getCause();
+        Thread thread = crashDetails.getThread();
+        AttributesBuilder attributesBuilder =
+                Attributes.builder()
+                        .put(SemanticAttributes.EXCEPTION_ESCAPED, true)
+                        .put(SemanticAttributes.THREAD_ID, thread.getId())
+                        .put(SemanticAttributes.THREAD_NAME, thread.getName())
+                        .put(SemanticAttributes.EXCEPTION_MESSAGE, throwable.getMessage())
+                        .put(SemanticAttributes.EXCEPTION_STACKTRACE, stackTraceToString(throwable))
+                        .put(SemanticAttributes.EXCEPTION_TYPE, throwable.getClass().getName());
+
+        for (AttributesExtractor<CrashDetails, Void> extractor : additionalExtractors) {
+            extractor.onStart(attributesBuilder, Context.current(), crashDetails);
+        }
+
+        crashReporter.logRecordBuilder().setAllAttributes(attributesBuilder.build()).emit();
```

Comment:
One bit that's non-obvious with this switch of signal types is that the attributes that might currently be applied from the `GlobalAttributesSpanAppender` are no longer applicable, so those attributes will be missing. I think that sort of breaking change is fine now, but I don't want us to lose sight of that. 

Would you mind opening an issue to track it and brainstorm solutions?

Reply:
Oh, and from a RUM standpoint, most importantly, I think this change, as it exists right now, prevents the session id from being attached to these crash LogRecords, right? I think that's pretty important.

Reply:
Good point, we should definitely allow for global attrs to be present in all types of signals. I've created this issue to work on it: https://github.com/open-telemetry/opentelemetry-android/issues/256

Reply:
I've created this PR to support global attrs in logs: https://github.com/open-telemetry/opentelemetry-android/pull/266